.gitignore

```diff
@@ -1,5 +1,5 @@
-F3DZEX_*/
-F3DEX2_*/
+build/
 .vscode
 *.exe
 *.bat
+custom.mk
```

Comment:
I don't think custom.mk should be gitignored. If somebody clones this repo for their own custom ucode project, they may accidentally not commit their custom.mk because of it.

Reply:
Changed